### PR TITLE
tabs vs spaces

### PR DESCRIPTION
**Pull request by fmessmer**
minor for consistency
(split this feature from #346)

### CODE REVIEW

**Comment by mathias-luedtke** [comment]
Looks fine

**Comment by NadiaHG** [comment]
LGTM

File: cob_bringup/drivers/mimic.launch [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <launch>
 
-  <!-- start sound -->
-  <node pkg="cob_mimic" type="mimic_node.py" name="mimic" cwd="node" respawn="false" />
+	<!-- start sound -->
```
Comment (NadiaHG) [inline cob_bringup/drivers/mimic.launch]: can you please also modify the comment?

**Comment by fmessmer** [comment]
done...

**Comment by floweisshardt** [comment]
LGTM

**Pull request merged**